### PR TITLE
Fix image size

### PR DESCRIPTION
**Pull request by Madhu-1**
Fixes: #214 

@rootfs @gman0 PTAL, let me know it changes looks good, i will test it out.

### CODE REVIEW

File: pkg/rbd/rbd_util.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func createRBDImage(pOpts *rbdVolume, volSz int, adminID string, credentials map
 	}
 
 	image := pOpts.VolName
-	volSzGB := fmt.Sprintf("%dG", volSz)
+	volSzGB := fmt.Sprintf("%dM", volSz)
```
Comment (gman0) [inline pkg/rbd/rbd_util.go]: Shouldn't this be in MiB? Also, the name of the variable doesn't reflect its contents.

Comment (Madhu-1) [reply pkg/rbd/rbd_util.go]: missed this one, will fix it

**Comment by rootfs** [comment]
looks good, is it still DNM?

**Comment by Madhu-1** [comment]
@rootfs we need to make a decision on minimum volume size on cephfs https://github.com/ceph/ceph-csi/issues/214#issuecomment-468649045

still i need to test this PR.

**Comment by rootfs** [comment]
sure, I'll let @gman0 make the final decision

**Comment by gman0** [comment]
@Madhu-1 @rootfs for cephfs:
1. the correct solution would be to handle `LimitBytes`, because we're not pre-allocating volume space but merely specifying a soft limit for maximum bytes in a directory. Unfortunately, external-provisioner handles only `RequiredBytes` at the moment: https://github.com/kubernetes-csi/external-provisioner/blob/c42566f2229722b0184e45781be91f1e40b8c86c/pkg/controller/controller.go#L497-L504
2. keep cephfs handling the size as it was before
3. a compromise: round up to 1MiB as is done in this PR, but still accept `0` as size

I'd go with option (2) because it's the closest to reality (in regards to ceph and csi spec) we can get with current state of external-provisioner.

**Comment by Madhu-1** [comment]
@gman0 reverted back the changes for ceph-fs PTAL

tested for rbd and working  fine

File: pkg/cephfs/volume.go [hunk 2]
```diff
@@ -71,10 +71,8 @@ func createVolume(volOptions *volumeOptions, adminCr *credentials, volID volumeI
 		return err
 	}
 
-	if bytesQuota > 0 {
-		if err := setVolumeAttribute(volRootCreating, "ceph.quota.max_bytes", fmt.Sprintf("%d", bytesQuota)); err != nil {
-			return err
-		}
+	if err := setVolumeAttribute(volRootCreating, "ceph.quota.max_bytes", fmt.Sprintf("%d", bytesQuota)); err != nil {
```
[the remark this answers is not in the document]
Comment (Madhu-1) [reply pkg/cephfs/volume.go]: yeah agreed, but if a user creates a PVC he sees the volume size mismatch in PVC describe and cephfs volume with current setup.

**Comment by Madhu-1** [comment]
@gman0 Mergify is failing can we merge this one manually?

**Pull request merged**